charts/enterprise/kubeapps/values.yaml

```diff
@@ -1,27 +1,68 @@
 image:
   pullPolicy: IfNotPresent
-  repository: tccr.io/tccr/alpine
-  tag: latest@sha256:ade0065e19edaa4f6903d464ee70605111a48394536deb94f31b661264704558
-manifestManager:
-  enabled: false
+  repository: ghcr.io/xstar97/my-proxy-service
+  tag: v0.0.1@sha256:2ef9bb1b94f24966e9e52bc398e9c1eb52f1582f22ffe2ebc3629cd2977e49c4
+
 operator:
   register: false
+  
 portal:
   open:
-    enabled: false
+    enabled: true
+    
+persistence:
+  token:
+    type: projected
+    mountPath: /service-account
+    sources:
+      - serviceAccountToken:
+          expirationSeconds: 3600
+          path: token
+          
+podOptions:
+  automountServiceAccountToken: true
+    
 workload:
-  main:
-    enabled: false
+  tcopen:
+    enabled: true
+    expandObjectName: false
+    type: Deployment
+    podSpec:
+      containers:
+        container-name:
+          env:
+            PORT: 3000    
+          args:
+            - --api-file /service-account/token
+            - --proxy-target http://tckubeapps:8080
+            - --auth-token-header authorization
```

Comment:
```suggestion
          env:
            PORT: 3000    
            ARGS:  --api-file /service-account/token --proxy-target http://tckubeapps:8080 --auth-token-header authorization
```

Reply:
you cant use the args without passing the go tool as the first one....
using the ARGS env variable allows you to set the args directly

Reply:
Maybe rename it to not conflict with ARGS?
Because args and envs are something else entirely.

Maybe split those -- args into seperate env vars?